coriolis_openstack_utils/actions/network_actions.py

```diff
@@ -283,3 +284,90 @@ def execute_operations(self):
             'dest_tenant_id': self.payload['dest_tenant_id']}
 
         return dest_network
+
+
+class RouterCreationAction(base.BaseAction):
+    """ Action class for replicating Neutron routers.
+    param action_payload: dict(): payload (params) for the action
+    must contain 'src_router_id'
+    """
+
+    action_type = base.ACTION_TYPE_CHECK_CREATE_ROUTER
+    NEW_ROUTER_DESCRIPTION = (
```

Comment:
Please rename this as `NEW_ROUTER_DESCRIPTION_FORMAT` to be clear(er) on what it's about.